modules/shared/php/options-utilities.php

```diff
@@ -0,0 +1,55 @@
+<?php
+/**
+ * class OptionsUtilities
+ *
+ * @desc Utility methods for initializing a module after first install
+ */
+
+namespace VIPWorkflow\Modules\Shared\PHP;
+
+use stdClass;
+use VIPWorkflow\VIP_Workflow;
+
+class OptionsUtilities {
```

Comment:
This is a set of functional utilities to manage options like `loaded_once` that's a bit less magic than before.